src/core/bitop.d

```diff
@@ -72,20 +72,18 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
+{
 /// ditto
+int bsf(ulong v) pure;
+}
+else version(D_LP64) { } else
```

Comment:
There is no `D_LP32` standard version. `size_t` is 32 bit unless `D_LP64` is set.